image/templates/helm/stackrox-secured-cluster/templates/sensor-netpol.yaml

```diff
@@ -26,6 +26,11 @@ spec:
       - podSelector:
           matchLabels:
             app: scanner
+{{ if ._rox.scannerV4._indexerEnabled }}
```

Comment:
Q: I am thinking about the situation where central and sensor are deployed in the same namespace and on sensor side we use
```
scanner:
  disable: false
scannerV4:
  disable: false
```
to enable local scanners. The chart would not deploy indexer, because an indexer is already deployed alongside central. In other words `._rox.scannerV4._indexerEnabled` would be false and we end up not adding scanner-v4-indexer to this network policy.
Is this OK and expected?

In case indexer has been deployed by the central chart, it would not try to talk to sensor even though a sensor is running in the same namespace?

Reply:
Correct, we don't expect Scanner V4 to reach out to Sensor in this scenario, though Sensor should be allowed to talk to Scanner V4. How would you recommend reconciling this? I'm also ok with just allowing the traffic, even though we don't expect it

Reply:
Indexer should only reach out to Sensor when deployed as a SecuredCluster Service

Reply:
See https://github.com/stackrox/stackrox/blob/master/image/templates/helm/shared/config-templates/scanner-v4/indexer-config.yaml.tpl